tests/config.py

```diff
@@ -5,5 +5,6 @@
 
 import os
 
-VALIDATOR_UI_URL = os.environ.get('FILENAME', 'http://localhost:8010')
-DIFFER_UI_URL = os.environ.get('FILENAME', 'http://localhost:8030')
+VALIDATOR_UI_URL = os.environ.get('VALIDATOR_UI_URL', 'http://localhost:8010')
```

Comment:
can we please rename this ENV variable into RDF_VALIDATOR ... , both here in the config.py and in the .env file